src/app/app-routing.module.ts

```diff
@@ -1,10 +1,17 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import {DashboardComponent} from "./side-bar/dashboard/dashboard.component";
```

Comment:
Single Quotation here

Reply:
I have resolved it la......................Thank you